app/controllers/onboardings_controller.rb

```diff
@@ -1,8 +0,0 @@
-class OnboardingsController < ApplicationController
-  before_action :authenticate_user!
-  before_action :set_cache_control_headers, only: %i[show]
-
-  def show
-    set_surrogate_key_header "onboarding-slideshow"
-  end
-end
```

Comment:
Since we cache aggressively, could removing an endpoint cause something in the JS app to try to continue fetching this?

For example, the user is running a cached version of the front-end app, will navigating around the application cause it to load the new JS?

Honest question, I don't actually know if that's a problem we have at all. 😄

Reply:
I think Jeremy and I were talking about this recently as well (after he saw confusing results when he added new policy info to the userData response). 


JS that's in packs (and included in the view templates for pages with a `<% pack` tag) _probably_ gets loaded correctly?

JS that's in the initializers (and included in the initial response, sitting at the same level as instanclick) _probably_ doesn't.

I'm unsure whether I understand this correctly though. 

I think the onboarding page falls into the "okay" category via the loading [in the view](https://github.com/forem/forem/blob/main/app/views/onboardings/show.html.erb#L19) - this code (that knows about the path to the route) gets loaded when you visit the page and not as part of the base app?

Reply:
I agree with @djuber 's assessment on this and it's been something that has been bothering me when I was trying to futilely cut down honeybadger noise. I'm quite certain the changes my changes here is fine but I can test on https://community.benhalpern.com/ first

Reply:
tested on staging. all is well.